src/data/code-samples/en/accessibility-name/swift-ui.md

```diff
@@ -0,0 +1,10 @@
+# Accessibility name - SwiftUI
+
+In SwiftUI, [`accessibilityLabel`](https://developer.apple.com/documentation/swiftui/view/accessibilitylabel(_:)-7rljm) property is used as accessibility name.
+
+```swift
+Link(destination: URL(string: "https://appt.org/en/")!, label: {
```

Comment:
Can you create an example of a `Button` with an icon/image e.g. search icon, and then set a11y label to `Search`